src/main/scala/cromwell/engine/db/slick/ExecutionComponent.scala

```diff
@@ -84,6 +85,27 @@ trait ExecutionComponent {
       if workflowExecution.workflowExecutionUuid === workflowExecutionUuid
     } yield execution)
 
+  val executionsByWorkflowExecutionIdAndCallKey = Compiled(
```

Comment:
Given our database constraints it won't return more than one, but it could return zero.  Agreed on the name, I'll change that.